frame/bonded-finance/README.md

```diff
@@ -0,0 +1,17 @@
+# Bonded Finance Pallet
+
+## Overview
+
+A simple pallet providing means of submitting bond offers.
+
+## Interface
+
+This pallet implements the `BondedFinance` trait from `composable-traits`.
+
+## Dispatchable Functions
+
+- `offer` - Register a new bond offer, allowing use to later bond it.
```

Comment:
oh, dislike copy pasted, I would rather be tasked to generate clean docs from Rust doc comments;)